hyperbolic-rs/preprocess.py

```diff
@@ -43,23 +43,30 @@
                                                             'this threshold will be clamped to min_distance')
 
 
-def map_item_ids_to_sequential_ids(samples):
+def map_raw_ids_to_sequential_ids(samples):
     """
     For each unique user or item id, this function creates a mapping to a sequence of number starting in 0.
     This will be the index of the embeddings in the model.
 
+    Items ids will be from 0 to n_items - 1.
+    Users ids will be from n_items to n_items + n_users - 1
+    This condition is required to later build the distance matrix
```

Comment:
You probably want to generalize this to more types, and a varying number of types?

Reply:
That is done in the new PR